tests/store/_unity_catalog/oss_registry/test_oss_unity_catalog_rest_store.py

```diff
@@ -0,0 +1,80 @@
+import json
+from unittest import mock
+
+import pytest
+
+from mlflow.entities.model_registry import RegisteredModelTag
+from mlflow.protos.unity_catalog_oss_messages_pb2 import (
+    CreateRegisteredModel,
+    RegisteredModelInfo,
+    TagKeyValue,
+)
+from mlflow.store._unity_catalog.oss_registry.oss_rest_store import OssUnityCatalogStore
+from mlflow.utils.proto_json_utils import message_to_json
+
+from tests.helper_functions import mock_http_200
+from tests.store._unity_catalog.conftest import _REGISTRY_HOST_CREDS
+
+
+@pytest.fixture
+def store(mock_databricks_uc_oss_host_creds):
+    with mock.patch("mlflow.utils.databricks_utils.get_databricks_host_creds"):
+        return OssUnityCatalogStore(store_uri="databricks-uc")
+
+
+def _args(endpoint, method, json_body, host_creds, extra_headers):
+    res = {
+        "host_creds": host_creds,
+        "endpoint": f"/api/2.0/unity-catalog/{endpoint}",
+        "method": method,
+        "extra_headers": extra_headers,
+    }
+    if extra_headers is None:
+        del res["extra_headers"]
+    if method == "GET":
+        res["params"] = json.loads(json_body)
+    else:
+        res["json"] = json.loads(json_body)
+    return res
+
+
+def _verify_requests(
+    http_request,
+    endpoint,
+    method,
+    proto_message,
+    host_creds=_REGISTRY_HOST_CREDS,
+    extra_headers=None,
+):
+    json_body = message_to_json(proto_message)
+    call_args = _args(endpoint, method, json_body, host_creds, extra_headers)
+    http_request.assert_any_call(**call_args)
+
+
+@mock_http_200
+def test_create_registered_model(mock_http, store):
+    description = "best model ever"
+    tags = [
+        RegisteredModelTag(key="key", value="value"),
+        RegisteredModelTag(key="anotherKey", value="some other value"),
+    ]
+    store.create_registered_model(
+        name="catalog_1.schema_1.model_1", description=description, tags=tags
+    )
+    _verify_requests(
+        mock_http,
+        "models",
+        "POST",
+        CreateRegisteredModel(
+            registered_model_info=RegisteredModelInfo(
+                name="model_1",
+                catalog_name="catalog_1",
+                schema_name="schema_1",
+                comment=description,
+                tags=[
```

Comment:
```suggestion
                tags=tags,
```

Reply:
Not the same type of tags: `tags: List[RegisteredModelTag]` while `RegisteredModelInfo.tags: List[TagKeyValue]`

Reply:
we can make a conversion like this
https://src.dev.databricks.com/mlflow/mlflow@8638ebe054938d3309808832ca022d7172c16566/-/blob/tests/store/_unity_catalog/model_registry/test_unity_catalog_rest_store.py?L184

Reply:
Sorry I missed that comment, will do right now